vars/withInfraPipeline.groovy

```diff
@@ -7,7 +7,7 @@ import uk.gov.hmcts.contino.Subscription
 import uk.gov.hmcts.contino.Environment
 import uk.gov.hmcts.pipeline.TeamConfig
 
-def call(String product, String component = null, Closure body) {
+def call(String product, String component = null, String expiresAfter = null, Closure body) {
```

Comment:
Don't modify the parameters as this is a breaking change, use configuration instead via the DSL:

https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/AppPipelineConfig.groovy